src/thor/astar.cc

```diff
@@ -463,12 +455,6 @@ void AStarPathAlgorithm::SetOrigin(GraphReader& graphreader,
 
     // DO NOT SET EdgeStatus - it messes up trivial paths with oneways
   }
-
-  // Set the origin timezone
-  if (closest_ni != nullptr && origin.has_date_time() && origin.date_time() == "current") {
-    origin.set_date_time(
-        DateTime::iso_date_time(DateTime::get_tz_db().from_index(closest_ni->timezone())));
-  }
```

Comment:
i gutted all the time setting stuff from all the algorithms becuse the new struct handles all that for you